gui/src/bitcoind.rs

```diff
@@ -206,8 +214,6 @@ impl Bitcoind {
 
         let mut process = command
             .args(&args)
-            .stdout(std::process::Stdio::piped())
-            .stderr(std::process::Stdio::piped())
```

Comment:
Wait, no. It's needed below. Either we remove the error logging below as well, or we keep the pipes and somehow remove them later? If we can't do that better to get rid of the `process.wait_with_output` entirely then.

Reply:
Could we use
```
.stdout(std::process::Stdio::null())
.stderr(std::process::Stdio::piped())
```
 as otherwise bitcoind's stdout will be written to Liana's logs. I think we can still pipe the stderr and use that to check for errors.

Reply:
It would still risk to get filled up and eventually stuck bitcoind